client/index.html

```diff
@@ -120,7 +120,7 @@ <h1 class="title">Connect</h1>
 								<div class="col-sm-3"></div>
 								<div class="col-sm-9">
 									<label class="tls">
-										<input type="checkbox" name="tls" <%= defaults.tls ? 'checked="checked"' : '' %>>
+										<input type="checkbox" name="tls" <%= defaults.tls ? 'checked="checked"' : '' %> <%= typeof(lockNetwork) !== "undefined" && lockNetwork ? 'disabled' : ''%>>
```

Comment:
(And actually, for checkboxes, no need to do `'checked="checked"'`, `"checked"` would be enough here)